setup.cfg

```diff
@@ -63,3 +66,14 @@ lines_between_types = 1
 lines_after_imports = 2
 known_first_party = build
 known_third_party = nox,pallets_sphinx_themes,pep517,pytest,setuptools,toml
+
+[coverage:run]
+omit =
```

Comment:
Why do we need to omit tests? I think we do care that test have good coverage. To ensure that we don't have tests that do nothing.

Reply:
Agreed. I have found more than a few bugs where I was shadowing a test or something because my test coverage was less than 100%.